docs/source/features/parallelisms.rst

```diff
@@ -5,15 +5,80 @@ Parallelisms
     :width: 800px
     :alt: Distributed Data Parallel
 
+Distributed Optimizer (ZeRO-1)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ZeRO-1 algorithm keeps model copies consistent by sharding the
+optimizer state between GPUs. During each optimization step, the
+parameter gradients are first summed and sharded (with a
+reduce-scatter collective), each GPU applies an optimization to its
+local shard of the parameters, and the updated parameter shards are
+broadcast to update all of the model copies (with an all-gather
+collective). This approach is attractive for large models since
+sharding the optimizer state can significantly reduce its memory
+footprint on individual GPUs. It also has, in theory, the same
+communication volume as DDP and its communication pattern has more
+opportunities for overlapping with compute.
+
+Enable Data Parallelism
+~~~~~~~~~~~~~~~~~~~~~~~
+
+DDP is the default parallelism scheme when NeMo is run on multiple
+GPUs. Enabling other parallelism schemes in the model configuration
+will decrease the size of the DP group, i.e. the number of identical
+model copies.
+
+In order to enable the distributed optimizer, set
+``model.optim.name=distributed_fused_adam`` in the model
+configuration. It can be configured with the following options:
+
+===========================  =========  ==================================================================================================================================
+Option                       Default    Description
+===========================  =========  ==================================================================================================================================
+``dtype``                    fp32       Optimizer state datatype
+``grad_sync_dtype``          ``dtype``  Gradient reduce-scatter datatype
+``overlap_grad_sync``        True       Overlap gradient reduce-scatter with compute
+``overlap_param_sync``       False      Overlap parameter all-gather with compute
+``bucket_cap_mb``            100        Buffer size (in MiB) for internal state and workspaces. Larger buckets have lower runtime overheads but may increase memory usage.
+``contiguous_param_buffer``  False      Allocate parameters as views into a large buffer. Helps avoid some data copies.
+``contiguous_grad_buffer``   True       Allocate parameter gradients as views into a large buffer. Helps avoid some data copies.
+===========================  =========  ==================================================================================================================================
+
+See the keyword arguments in `Apex DistributedFusedAdam <https://github.com/NVIDIA/apex/blob/master/apex/contrib/optimizers/distributed_fused_adam.py>`_ and `NeMo MegatronDistributedFusedAdam <https://github.com/NVIDIA/NeMo/blob/main/nemo/core/optim/distributed_adam.py>`_ for a full list of distributed optimizer options.
+
+Implement Data Parallelism
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+NeMo's DDP either uses PyTorch
```

Comment:
Change "Nemo's DDP either uses" to "The NeMo DDP uses either . . ."

Note: We avoid forming a possessive from a feature name, product name, or trademark. For example, we use NeMo rather than NeMo's or NVIDIA and not NVIDIA's.